models/amat2_psc_exp.h

```diff
@@ -240,7 +244,7 @@ class amat2_psc_exp : public Archiving_Node
     /** Resting threshold in mV
         (relative to resting potential).
         The real resting threshold is (E_L_+omega_).
-        Called omega in [3]. */
+        Called omega in [3]_. */
```

Comment:
I saw this change multiple times in this PR. And although it is consistent with the other changes, I'm not sure that this change is useful. The reason for my doubt is that this Doxygen style comment (`/** ... */`) will end up in the *developer* documentation, while the reference itself is defined in the *user* documentation.

The proper solution is most likely to
1. add/extend Doxygen class level documentation to point to the location of the user documentation for the model on readthedocs and mention that references are contained therein
1. make the references in the Doxygen comments throughout the model code be the same format as that of the rendered user documentation on readthedocs.